src/ApiStack.ts

```diff
@@ -48,13 +48,21 @@ export class ApiStack extends Stack {
     });
   }
 
+  /**
+   * Function that validates the incoming profile and forwards the request.
+   * @param idTable Table containing the relevant application-ids (api keys) related to a specific set of fields.
+   * @param devMode Wether or not devmode is enabled.
+   * @returns The personen lambda
+   */
   private personenFunction(idTable: Table, devMode: boolean) {
```

Comment:
Wil je devmode nog gebruiken voor specifieke dingen? Ik zie hem nog nergens gebruikt worden. Heb je die voor het geval dat ingebouwd?

Reply:
Die wordt gebruikt in de lambda

Reply:
Goede, zit in de env var, had ik over het hoofd gezien in deze PR context